wallet/src/wallet/mod.rs

```diff
@@ -63,8 +64,18 @@ pub enum WalletError {
     NoAccountFound(AccountId),
     #[error("No account found with index {0}")]
     NoAccountFoundWithIndex(U31),
+    #[error("Account with index {0} is not synced yet")]
+    AccountNotSyncedWithIndex(U31),
     #[error("Account with index {0} already exists")]
     AccountAlreadyExists(U31),
+    #[error("Cannot create a new account when last account is still empty")]
+    EmptyLastAccount,
+    #[error("Cannot create a new account when last account is still not in sync")]
+    LastAccountNotInSync,
+    #[error("The maximum number of accounts has been exceeded")]
+    MaxNumAccountsExceeded,
```

Comment:
This is the maximum number of accounts due to it being u31, so it's `1 << 32 - 1`, right? If so, please call it "AbsoluteMaxNumAccountsExceeded", and change the message to "The absolute maximum number of accounts has been exceeded {}", and in it put that value.

Reply:
You seem to have forgotten to write the maximum value in the error message.